docker-compose.yml

```diff
@@ -129,6 +127,7 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: admin
       CANDIG_OPA_URL: https://opa:8181
+      CANDIG_OPA_VERSION: rego_dev_playground
```

Comment:
I take it that this is to distinguish between the format that's being used here and what's currently being used in candig/katsu?  Let's see if we can come up with a better naming scheme than this

Reply:
Yes, it is used in katsu to distinguish between what's used in dycons and ours.